pkg/apis/externaldns/types.go

```diff
@@ -448,7 +446,7 @@ func App(cfg *Config) *kingpin.Application {
 	app.Flag("gloo-namespace", "The Gloo Proxy namespace; specify multiple times for multiple namespaces. (default: gloo-system)").Default("gloo-system").StringsVar(&cfg.GlooNamespaces)
 
 	// Flags related to Skipper RouteGroup
-	app.Flag("skipper-routegroup-groupversion", "The resource version for skipper routegroup").Default(source.DefaultRoutegroupVersion).StringVar(&cfg.SkipperRouteGroupVersion)
+	app.Flag("skipper-routegroup-groupversion", "The resource version for skipper routegroup").Default("zalando.org/v1").StringVar(&cfg.SkipperRouteGroupVersion)
```

Comment:
Nitpick - we already have 2 "zalando.org/v1" hardcodes (this one is 3rd):
https://github.com/kubernetes-sigs/external-dns/blob/a6e57713cebb57d05b207a3fc08678f28a781b5b/source/skipper_routegroup.go#L44
and
https://github.com/kubernetes-sigs/external-dns/blob/a6e57713cebb57d05b207a3fc08678f28a781b5b/pkg/apis/externaldns/types.go#L225

Maybe for the sake of consistency with the code around ("gloo-system" above seems to be an exception) use `defaultConfig.SkipperRouteGroupVersion` here instead, and `source.DefaultRoutegroupVersion` in `types.go:225` (and `types_test.go`), to keep only one?

Reply:
Done